src/gui.py

```diff
@@ -110,19 +104,45 @@ def on_prop_selected(self, selection: Gtk.TreeSelection):
         def on_btn_edit_clicked(self, button: Gtk.Button):
             prop = self.gui.get_selected_prop()
             
-            self.gui.entry_old_val.set_text(prop['val'])
-            self.gui.entry_new_val.set_text(prop['val'])
-            self.gui.win_edit.show_all()
-            self.gui.entry_new_val.grab_focus()
+            dialog = Gui.EditMessageDialog(self.gui, prop['val'])
+            result = dialog.run()
+            if result == Gtk.ResponseType.APPLY: # Continue
+                dialog.on_btn_edit_apply_clicked()
+            dialog.destroy()
 
-        def on_entry_new_val_activate(self, entry: Gtk.Entry):
-            self.gui.btn_edit_apply.clicked()
 
-        def on_btn_edit_apply_clicked(self, button: Gtk.Button):
+    class EditMessageDialog(Gtk.MessageDialog):
```

Comment:
Instead of having `grid_edit` be added to a new `GtkMessageDialog`, I just created the dialog directly in Glade and moved the grid into it, which means most of the dialog setup here is done in Glade directly, so for now I've kept everything in the GUI class.

I am a fan of splitting the code up, and that's something that I plan on doing very soon.